services/settings/pkg/config/config.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 	HTTP HTTP `yaml:"http"`
 	GRPC GRPC `yaml:"grpc"`
 
+	GRPCClientTLS  *shared.GRPCClientTLS  `yaml:"grpc_client_tls"`
+	GRPCServiceTLS *shared.GRPCServiceTLS `yaml:"grpc_service_tls"`
```

Comment:
move this to the `GRPC` struct as `yaml:"tls"`?

Reply:
done